docs/source/using/accessing-the-openshift-docker-registry.adoc

```diff
@@ -0,0 +1,25 @@
+= Accessing the OpenShift Docker Registry
+:icons:
+:toc: macro
+:toc-title:
+:toclevels: 1
+
+OpenShift provides an integrated docker registry which can be used as a local registry.
+
+This helps to speed up the local development as the images present in the registry can be used as imagestream for running an application in the OpenShift
```

Comment:
Need a full stop at the end of this end. One line is need saying "To achieve that, perform following steps (or similar lines)".